src/instance/ListFieldInstance.ts

```diff
@@ -123,6 +128,14 @@ export default class ListFieldInstance implements Pluggable, Child {
   private getFakeKeys() {
     return Array.from(this.model.model.keys.values()).map(key => ({ key, value: FAKE_KEY }));
   }
+
+  private formatInstanceReferences(references: string[], suggestions: string[]) {
+    return _.uniq(
```

Comment:
These few lines seem to be repeated a few time. Consider putting it in a common place.